### DISCUSSION TITLE
How to make visible Markdown

### DISCUSSION THREAD

**Discussion by rakibmdabusayed** [mrhrifat/mrhrifat#4] [Q&A] [answered]
Hey I saw that, in your repository the coding language show `Markdown`. But it's not working on mine. How to make visible it.

**Comment by mrhrifat** [comment, marked as answer]
To make visible `Markdown` on Repository.
- Create a file named `.gitattributes` on Root Directory.
- Then write this 2 line of code given below
```
*.md linguist-detectable=true
*.md linguist-documentation=false
```
- Commits your Code

That's it.
